felix/fv/bpf_dual_stack_test.go

```diff
@@ -299,6 +348,11 @@ func describeBPFDualStackTests(ctlbEnabled, ipv6Dataplane bool) bool {
 					cc.CheckConnectivity()
 					cc.ResetExpectations()
 
+					// Since we allow the IPv6 packets through IPv4 programs, a stale neighbor entry might get created
+					// when trying to reach w[0][0] from workloads in felix-1. This will impact subsequent
+					// tests. Hence cleaning up the neighbor entry.
```

Comment:
It should mention that this does not seem to be a problem on ubuntu 22+ but is on 20.